sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -178,6 +178,75 @@ In scenarios involving multiple scopes or a single scope with multiple key-value
 only the first occurrence of the key-value pair from the outermost scope will be recorded.
 However, when the same key is utilized both within a logging scope and directly in the log statement, the value specified in the log message template will take precedence.
 
+### Application Insights Custom Events
+
+In general, you should be able to use
+[ILogger](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger)
+for your logging needs. For users looking to log [Application Insights Custom
+Events](https://learn.microsoft.com/azure/azure-monitor/app/api-custom-events-metrics#trackevent),
+this can be accomplished by utilizing the `TrackEvent(string name,
+IReadOnlyList<KeyValuePair<string, object?>>? attributes = null)` API provided
+in the exporter.
+
+To begin, you need to pass an instance of the
+[LoggerFactory](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.loggerfactory)
+class to create an instance of `ApplicationInsightsEventLogger`. This instance
+can then be used to invoke the `TrackEvent` method for logging custom events.
+Below is an example of how you can implement this:
+
+```csharp
+// Example code to log custom events in Application Insights
```

Comment:
consider referencing the using in this code example. or may using the fully qualified name with the EventLogger class.

We don't consistently do this in our docs, but this class is in a different namespace that might not be as discoverable.